pkg/graveler/graveler.go

```diff
@@ -111,9 +108,22 @@ type MergeStrategy int
 const (
 	MergeStrategyNone MergeStrategy = iota
 	MergeStrategyDest
-	MergeStrategySource
+	MergeStrategySrc
+
+	MergeStrategyNoneStr     = "default"
+	MergeStrategyDestWinsStr = "dest-wins"
+	MergeStrategySrcWinsStr  = "source-wins"
+
+	MergeStrategyMetadataKey = "lakefs.merge.strategy"
 )
 
+// mergeStrategyString String representation for MergeStrategy consts. Pay attention to the order!
```

Comment:
Why does the order matter?

Reply:
Because the const indexes are coupled to the string representation array